src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationRequest.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.entity
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.vladmihalcea.hibernate.type.json.JsonType
+import com.vladmihalcea.hibernate.type.json.internal.JacksonUtil
+import org.hibernate.annotations.Type
+import org.hibernate.annotations.TypeDef
+import org.hibernate.annotations.TypeDefs
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table
+@TypeDefs(
+  TypeDef(name = "json", typeClass = JsonType::class)
+)
+data class CalculationRequest(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long = -1,
+  val calculationReference: UUID = UUID.randomUUID(),
+  val prisonerId: String = "",
```

Comment:
Is null better than empty strings "" for default values? Or are the columns not nullable?

Reply:
They are actually not nullable.. will annotate with @NotNull